lib/DBDefs/Default.pm

```diff
@@ -271,6 +271,15 @@ sub CACHE_MANAGER_OPTIONS {
     return \%CACHE_MANAGER_OPTIONS
 }
 
+# Sets the TTL for entities stored in memcached, in seconds. On slave servers,
+# this is set to 1 hour by default, to mitigate MBS-8726. On standalone
+# servers, this is set to 0 (meaning no expiration is set), because cache
+# invalidation is already handled by the server in that case.
+sub ENTITY_CACHE_TTL {
+    return 3600 if shift->REPLICATION_TYPE == RT_SLAVE;
+    return 0;
+}
```

Comment:
Redis is easier to scale, as it comes with master/slave options. This is the main reason i asked if we can move to it, performance-wise i don't think it will make much difference anyway. Redis has much more options and since it is already used, as you said, it will actually reduce number of dependencies.